src/messaging/src/index.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import * as v1 from './v1';
+const FcmServiceClient = v1.FcmServiceClient;
+type FcmServiceClient = v1.FcmServiceClient;
+export {v1, FcmServiceClient};
```

Comment:
We need a way to suppress these lint errors. We can either ask the REGAPIC folks to fix it so the generated code has no lint errors, or update our eslint configuration to avoid lint checking these files.

Reply:
Updated eslint config for now.